tf-modules/report-granules/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const get = require('lodash.get');
+const has = require('lodash.has');
+
+const {
+  isSnsEvent,
+  getSnsEventMessageObject
+} = require('@cumulus/common/sns-event');
+const Granule = require('@cumulus/api/models/granules');
+
+/**
+ * Check if message contains granules.
+ * Checks locations expected by sns2elasticsearch
```

Comment:
No. I'm thinking about deleting this function. It filters messages for ones that have one of the two fields, because that's what `models/granules.createGranulesFromSns` does. But since we call that in the next function in the chain, it's pretty redundant. It also duplicates assumptions about the message from that function.